src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+    }
```

Comment:
we don't need to declare default public constructor, unless we have other constructors